guardian/grid
# Get S3 Watcher running locally

## What does this change?

Update S3 Watcher development scripts to run locally. Main points were removing the handling for ini config files, and configuring for interacting with localstack.

It's not super pretty (there's definitely some scope for DRYing) but it at least now works, and you can test out s3watcher locally.

## How should a reviewer test this change?

Set up and have the grid running locally. Then, in another terminal, open s3watcher and run the setup script and then the run-local script. There should be no errors in the terminal, and the image should be uploaded into your local grid.

## How can success be measured?

Changes and upgrades for s3 watcher can be tested locally

## Who should look at this?


## Tested? Documented?
